### DISCUSSION TITLE
View fields of linked objects in the associated object tables and views

### DISCUSSION THREAD

**Discussion by rob-luke** [twentyhq/twenty#9074] [Ideas]
## Scope & Context

It would be great if you could see fields of linked objects in the associate object table and view.

For example, when viewing an opportunity, I would like to see the country of the associated company without clicking in to the company.

## Current behavior

Currently, linked object fields are not displayed in the associated object table or view. 

For example:
- When viewing an opportunity, the country of the associated company is not shown unless you navigate to the company details.
- Similarly, the email of the person linked to an opportunity is not shown in the opportunity view.

This requires users to perform additional navigation to gather essential context for linked objects.

## Expected behavior
The fields of linked objects should be visible in the associated object table and view without requiring additional navigation.

For example:
- When viewing an opportunity, the country field from the associated company should be displayed as part of the opportunity details or table.
- Similarly, the email of the associated person should be shown in the opportunity fields.

This functionality would improve usability and streamline access to key information for users.

## Technical inputs

I tried to search issues and discord for this before posting, but I am not sure of the technical terms to describe this features, so apologies if this is listed elsewhere.

**Comment by rob-luke** [comment]
I think this might be the same as described here https://github.com/twentyhq/twenty/discussions/3645 ?

**Comment by FelixMalfait** [comment]
Hey @rob-luke great feedback again. This is something we were discussing again with @Bonapara yesterday. We'll tackle it in Q2 next year I think because Q1 is already a bit busy (workflows, permissions, and most importantly many quality improvements)

**Comment by rob-luke** [comment]
Thanks @FelixMalfait , glad to know it’s on the roadmap.

**Comment by BOHEUS** [comment]
@rob-luke you can see related record's fields in record details (or that's not what you're looking for, it should be available in Q3 https://github.com/twentyhq/core-team-issues/issues/214)
<img width="1680" height="1014" alt="Screenshot From 2026-03-18 10-24-37" src="https://github.com/user-attachments/assets/4bc2566a-41cc-48b2-ac16-a1f597e75bb3" />
